### PR TITLE
Update in response to more IRSG review

### PR DESCRIPTION
**Pull request by wbl**

### CODE REVIEW

File: draft-irtf-cfrg-spake2.xml [hunk 1]
```diff
@@ -175,14 +179,14 @@
 
       <section title="SPAKE2" anchor="spake2">
         <t>To begin, A picks x randomly and uniformly from the integers in [0,p),
-            and calculates X=x*P and S=w*M+X, then transmits pA=S to B.</t>
+            and calculates X=x*P and pA=w*M+X, then transmits pA to B.</t>
```
Comment (chris-wood) [inline draft-irtf-cfrg-spake2.xml:182]: This seems less clear than what was there previously, so I'd personally revert this. But, 🤷‍♂️

File: draft-irtf-cfrg-spake2.xml [hunk 2]
```diff
@@ -143,8 +143,12 @@
 
       <section title="Protocol Flow" anchor="flow">
         <t>SPAKE2 is a two round protocol, wherein the first round establishes a shared secret 
-          between A and B, and the second round servers as key confirmation. Prior to invocation, A and B are provisioned with
-          information such as the input password needed to run the protocol. 
+          between A and B, and the second round server as key confirmation. Prior to invocation, A and B are provisioned with
```
Comment (kaduk) [inline draft-irtf-cfrg-spake2.xml:146]:
```suggestion
          between A and B, and the second round serves as key confirmation. Prior to invocation, A and B are provisioned with
```

File: draft-irtf-cfrg-spake2.xml [hunk 3]
```diff
@@ -169,20 +173,22 @@
   (compute cA)  |        cA        |
                 |----------------->|
                 |        cB        | (compute cB)
+		|                  | (check cA)
                 |<-----------------|
+  (check cB)	|                  |
```
Comment (kaduk) [inline draft-irtf-cfrg-spake2.xml:178]: Is there a tabs/spaces mismatch here?

File: draft-irtf-cfrg-spake2.xml [hunk 4]
```diff
@@ -222,14 +228,15 @@
...
+	128 bits in length.</t>
 
         <t>Both endpoints use Ka to derive subsequent MAC keys for key confirmation messages.
             Specifically, let KcA and KcB be the MAC keys used by A and B, respectively.
-            A and B compute them as KcA || KcB = KDF(Ka, nil, "ConfirmationKeys" || AAD), where AAD
+            A and B compute them as KcA || KcB = KDF(Ka, nil, "ConfirmationKeys" || AAD, L), where AAD
             is the associated data each given to each endpoint, or nil if none was provided.
             The length of each of KcA and KcB is equal to half of the underlying hash output length, e.g.,
-            |KcA| = |KcB| = 128 bits for HKDF(SHA256). </t>
+            |KcA| = |KcB| = 128 bits for HKDF(SHA256), and L=256 bits. </t>
```
Comment (kaduk) [inline draft-irtf-cfrg-spake2.xml:239]:
```suggestion
            |KcA| = |KcB| = 128 bits for HKDF(SHA256), with L=256 bits. </t>
```

**Review by kaduk — COMMENTED** [review]
I can see why Chris finds the pA form less readable than S, but it may be a personal preference thing.
Suggestions inline, but this seems fine overall.

**Comment by chris-wood** [comment]
Oh it's definitely a style thing :) feel free to ignore me!

File: draft-irtf-cfrg-spake2.xml (hunk 3 above)
Comment (wbl) [reply draft-irtf-cfrg-spake2.xml:178]: yup. it looked right in emacs!

**Pull request merged**